include/deal.II/base/function_parser.h

```diff
@@ -246,6 +246,12 @@ class FunctionParser : public AutoDerivativeFunction<dim>
                                                      ",t",
                  const double h = 1e-8);
 
+  /**
+   * Copy constructor. Objects of this type can not be copied, and
+   * consequently this constructor is deleted.
+   */
+  FunctionParse(const FunctionParser &) = delete;
```

Comment:
```suggestion
  FunctionParser(const FunctionParser &) = delete;
```

Reply:
That's what happens if you're working on an unplugged laptop and don't want to compile the entire library...